dom/test/uix/dom/server_test.cljc

```diff
@@ -482,4 +488,8 @@
 
      (deftest test-vector-children
        (is (= (render ($ :div [($ :span) ($ :span)]))
-              "<div><span></span><span></span></div>")))))
+              "<div><span></span><span></span></div>")))
+
+     (deftest test-self-closing-tags
+       (is (= (render ($ :div ($ :img) ($ :br)))
+              "<div><img/><br/></div>")))))
```

Comment:
I think br is a special case where it should be just `<br>`. At least this is what I get when doing playground. I might be wrong. I'll check react SSR and come back

Reply:
there's an additional test above that runs against SSRed output from React itself, so I'm matching whatever React is producing